test/runtests.jl

```diff
@@ -1,5 +1,8 @@
 using Libm
 using Base.Test
 
-# write your own tests here
-@test 1 == 2
+@testset "Libm" begin
```

Comment:
Best to make the test set specific to a particular function or group of related functions.